app/controllers/questions_controller.rb

```diff
@@ -17,6 +17,8 @@ def create
 
   def show
     @question = Question.find(params[:id])
+    @solution = @question.solutions.build
```

Comment:
ここで作成した`@solution`が `@solutions` に含まれてしまっていることが原因のようです。

```suggestion
    @solution = Solution.new
```

このように修正すれば解決するでしょう。これでも同じエラーが出る場合は，コンソールから`Solution.destroy_all`で異常な回答を全部削除しましょう。